server/src/database/migrations/sqlite/001-setup.sql

```diff
@@ -18,15 +18,15 @@ CREATE TABLE containers (
   url           VARCHAR(255) NOT NULL,
   user_id       CHAR(8) NOT NULL,
   blob          TEXT,
-  FOREIGN KEY (user_id) REFERENCES users(id)
+  FOREIGN KEY (user_id) REFERENCES users(id) ON DELETE CASCADE
```

Comment:
quite dangerous to delete database values of container if user gets deleted, will lead to unknown instances continuing running without the program knowing?

Reply:
Yea that is not good, should not do this then I guess